core/unit_test/TestExecSpaceThreadSafety.hpp

```diff
@@ -77,6 +77,10 @@ void run_exec_space_thread_safety_range() {
 }
 
 TEST(TEST_CATEGORY, exec_space_thread_safety_range) {
+#ifdef KOKKOS_ENABLE_OPENACC  // FIXME_OPENACC
```

Comment:
Why disabled here? As I know, exec_space_thread_safety_range will work for the OpenACC backend.

Reply:
```
[ RUN      ] openacc.exec_space_thread_safety_range
3: Kokkos::Impl::SharedAllocationRecord '' failed decrement count = 0
3: terminate called after throwing an instance of 'std::runtime_error'
3:   what():  Kokkos::Impl::SharedAllocationRecord failed decrement count
```
If you could investigate this further that would be great. I just want the CI to pass again without investigating the cause of the failures too much.

Reply:
The test checks if kernels submitted to the same execution space instance from multiple threads are enqueued and don't run concurrently.